libs/api-services/src/lib/config.ts

```diff
@@ -0,0 +1,18 @@
+const getEnvOrThrow = (name: string) => {
```

Comment:
I get the idea of breaking out code in the 80-20 approach, but it feels like these env variable are specific to the backend (i.e. frontend will never need this as required). Are we saying that's fine because of the nature on how are docker images are built they'll always be sharing the same secrets?

Reply:
This exists in api-services, are we using api-services in the frontend?

Reply:
ah we aren't didn't see that it was being tacked on there

Reply:
Additionally, if we move this back to the backend, how do we use it from the library?